build/verify-features.ts

```diff
@@ -0,0 +1,47 @@
+import path from 'node:path';
+import {readdirSync as readDirSync, readFileSync} from 'node:fs';
+
+import {getFeaturesMeta} from './readme-parser';
+
+const featuresDirContents = readDirSync(path.join(__dirname, '../source/features/'));
+const refinedGithubTs = readFileSync(path.join(__dirname, '../source/refined-github.ts')).toString('utf-8');
```

Comment:
You can export these two lines in their own functions (and reuse them in `readme-parser` too)

https://github.com/sindresorhus/refined-github/blob/cb722b0639a0bf5f546b036ff95d9f4c26930517/build/readme-parser.ts#L56-L57

```suggestion
const importedFeatures = getImportedFeatures();
```